glib/src/auto/enums.rs

```diff
@@ -674,3 +674,133 @@ impl ErrorDomain for UriError {
         }
     }
 }
+
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+#[non_exhaustive]
+#[doc(alias = "GVariantClass")]
+pub enum VariantClass {
+    #[doc(alias = "G_VARIANT_CLASS_BOOLEAN")]
+    Boolean,
+    #[doc(alias = "G_VARIANT_CLASS_BYTE")]
+    Byte,
+    #[doc(alias = "G_VARIANT_CLASS_INT16")]
+    Int16,
+    #[doc(alias = "G_VARIANT_CLASS_UINT16")]
+    Uint16,
+    #[doc(alias = "G_VARIANT_CLASS_INT32")]
+    Int32,
+    #[doc(alias = "G_VARIANT_CLASS_UINT32")]
+    Uint32,
+    #[doc(alias = "G_VARIANT_CLASS_INT64")]
+    Int64,
+    #[doc(alias = "G_VARIANT_CLASS_UINT64")]
+    Uint64,
+    #[doc(alias = "G_VARIANT_CLASS_HANDLE")]
+    Handle,
+    #[doc(alias = "G_VARIANT_CLASS_DOUBLE")]
+    Double,
+    #[doc(alias = "G_VARIANT_CLASS_STRING")]
+    String,
+    #[doc(alias = "G_VARIANT_CLASS_OBJECT_PATH")]
+    ObjectPath,
+    #[doc(alias = "G_VARIANT_CLASS_SIGNATURE")]
+    Signature,
+    #[doc(alias = "G_VARIANT_CLASS_VARIANT")]
+    Variant,
+    #[doc(alias = "G_VARIANT_CLASS_MAYBE")]
+    Maybe,
+    #[doc(alias = "G_VARIANT_CLASS_ARRAY")]
+    Array,
+    #[doc(alias = "G_VARIANT_CLASS_TUPLE")]
+    Tuple,
+    #[doc(alias = "G_VARIANT_CLASS_DICT_ENTRY")]
+    DictEntry,
+    #[doc(hidden)]
+    __Unknown(i32),
+}
+
+impl fmt::Display for VariantClass {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "VariantClass::{}",
+            match *self {
+                Self::Boolean => "Boolean",
+                Self::Byte => "Byte",
+                Self::Int16 => "Int16",
+                Self::Uint16 => "Uint16",
+                Self::Int32 => "Int32",
+                Self::Uint32 => "Uint32",
+                Self::Int64 => "Int64",
+                Self::Uint64 => "Uint64",
+                Self::Handle => "Handle",
+                Self::Double => "Double",
+                Self::String => "String",
+                Self::ObjectPath => "ObjectPath",
+                Self::Signature => "Signature",
+                Self::Variant => "Variant",
+                Self::Maybe => "Maybe",
+                Self::Array => "Array",
+                Self::Tuple => "Tuple",
+                Self::DictEntry => "DictEntry",
+                _ => "Unknown",
```

Comment:
Can you instead bind `Self::_Unknown(_)` please? It's especially important in case we add new variants: we'll be sure to not forget to add them into the pattern matching thanks to it.

Reply:
This is autogenerated code

Reply:
Oh. Ok then it's good, I'll open an issue on `gir` because it's not great.